### PR TITLE
Problem: The address and the private keys should be derived from the mnemonics in integration test

### LINKED ISSUE #240: Problem: The address and the private keys should be derived from the mnemonics in integration test
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by damoncro**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

Close #240 

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [x] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [x] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [x] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: integration_tests/utils.py [hunk 1]
```diff
@@ -25,11 +25,13 @@
...
-KEYS = {
-    "validator": os.getenv("VALIDATOR_KEY"),
-    "community": os.getenv("COMMUNITY_KEY"),
+Account.enable_unaudited_hdwallet_features()
+ACCOUNTS = {
+    "validator": Account.from_mnemonic(os.getenv("VALIDATOR1_MNEMONIC")),
+    "community": Account.from_mnemonic(os.getenv("COMMUNITY_MNEMONIC")),
 }
-ADDRS = {name: Account.from_key(key).address for name, key in KEYS.items()}
+KEYS = {name: account.key.hex().replace("0x", "") for name, account in ACCOUNTS.items()}
```
Comment (yihuang) [inline integration_tests/utils.py:33]:
```suggestion
KEYS = {name: binascii.hexlify(account.key) for name, account in ACCOUNTS.items()}
```

File: scripts/start-geth [hunk 2]
```diff
@@ -24,14 +24,26 @@ $PASSWORD
 EOF
 
 # import validator key
+validator_key=$(python -c """
```
Comment (yihuang) [inline scripts/start-geth:27]: But can we run `start-geth` directly from cli after this?

Comment (damoncro) [reply scripts/start-geth:27]: Yes, we can, in nix-shell.

Comment (yihuang) [reply scripts/start-geth:27]: where is the `.env` sourced?

Comment (yihuang) [reply scripts/start-geth:27]: Or one should source it by himself, or put it in shell profile?

File: integration_tests/utils.py (hunk 1 above)
Comment (damoncro) [reply integration_tests/utils.py:33]: `hexlify` returns a bytes, should we need it be string?

File: scripts/start-geth (hunk 2 above)
Comment (damoncro) [reply scripts/start-geth:27]: Yes, in nix shell, source in `start-geth` defined in `script.nix`. For the script itself, no source.

Comment (damoncro) [reply scripts/start-geth:27]: It is difficult to add it in script, because the `.env` relative path depends on the user's working directory.

Comment (damoncro) [reply scripts/start-geth:27]: Or use full path, but full path also can be changed.

File: integration_tests/utils.py (hunk 1 above)
Comment (yihuang) [reply integration_tests/utils.py:33]: I think `account.key` alone should be good here, in the end, it's passed to `Account.from_key`, which accepts `bytes`.

Comment (damoncro) [reply integration_tests/utils.py]: You are right, updated. But it it still necessary in `start-geth` which expects string.

**Review by yihuang — APPROVED** [review]

**Pull request merged**